web/src/beta/components/fields/LocationField/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCallback, useEffect, useState } from "react";
+
+import { LatLng } from "@reearth/beta/utils/value";
+import { styled } from "@reearth/services/theme";
+
+import Property from "..";
+import NumberInput from "../common/NumberInput";
+
+type Props = {
+  name?: string;
```

Comment:
Should there be a disabled state for the field?

Reply:
I don't think so it is not exist in design